docs/examples/cli/README.md

```diff
@@ -5,70 +5,123 @@ The deepset Cloud CLI is a command-line interface tool that you can use to inter
 To install the deepset Cloud CLI, use `pip`:
 
 ```shell
-pip3 install deepset-cloud-sdk
+pip install deepset-cloud-sdk
 ```
 ## Configuration
 Before using the deepset Cloud CLI, log in and provide your credentials. You can do this by running the command:
 
+On MacOS and Linux:
+
 ```shell
 deepset-cloud login
 ```
+On Windows:
+
+```shell
+python -m deepset_cloud_sdk.cli login
```

Comment:
Theoretically we can also make it work on windows by specifying the correcnt PATH to the cli. Let's keep it like this for now